src/test/java/com/tratif/pdfgen/document/DocumentTest.java

```diff
@@ -0,0 +1,38 @@
+package com.tratif.pdfgen.document;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.io.Reader;
+import java.io.StringReader;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DocumentTest {
+
+    @Rule
+    ExpectedException exception = ExpectedException.none();
+
+    @Test
+    public void hasProperContentFromString() {
+        assertThat(Document.fromStaticHtml("<h1>Hello world</h1>").getContent())
```

Comment:
`getContent` is not needed for  production code and was added only to test it. It's an antipattern.
This test doesn't verify the real functionality of  DocumentBuilder.
`getContent` should be removed. Then you can do one of the following:
 - use `.html()` or `.pdf()` of DocumentBuilder in these tests
 - create utility class for reading from various sources (streams, readers, urls etc) and test it separately